### ISSUE TITLE
Incorrect rfc3164 message parsing

### ISSUE THREAD

**Issue by iatsiuk** [solzimer/nsyslog-parser#2]
input:

```
<30>Aug  4 16:53:13 pinger[1334]: 64 bytes from 87.250.250.242: seq=69 ttl=37 time=33.918 ms\n
```

output:

```
{ originalMessage: '<30>Aug  4 16:53:13 pinger[1334]: 64 bytes from 87.250.250.242: seq=69 ttl=37 time=33.918 ms\n',
pri: '<30>',
prival: 30,
facilityval: 3,
levelval: 6,
facility: 'daemon',
level: 'info',
type: 'BSD',
ts: 2018-08-04T16:53:13.000Z,
message: 'pinger[1334]: 64 bytes from 87.250.250.242: seq=69 ttl=37 time=33.918 ms\n',
chain: [],
host: '',
fields: [],
header: '<30>Aug  4 16:53:13 ' }
```

Check `message` property. This log entry produced by docker.

**Comment by iatsiuk** [comment]
And this is output from [squeeks/glossy](https://github.com/squeeks/glossy) library:

```
{
originalMessage: '<30>Aug  4 16:57:38 pinger[1334]: 64 bytes from 87.250.250.242: seq=334 ttl=37 time=35.967 ms\n',
prival: 30,
facilityID: 3,
severityID: 6,
facility: 'daemon',
severity: 'info',
type: 'RFC3164',
time: 2018-08-04T16:57:38.000Z,
host: 'pinger[1334]:',
message: '64 bytes from 87.250.250.242: seq=334 ttl=37 time=35.967 ms\n' 
} 
```

**Comment by solzimer** [comment]
Hi @iatsiuk 

The bug has been corrected in version [0.8.3](https://www.npmjs.com/package/nsyslog-parser)

Now the output is:

```
{ originalMessage: '<30>Aug  4 16:53:13 pinger[1334]: 64 bytes from 87.250.250.242: seq=69 ttl=37 time=33.918 ms\n',
  pri: '<30>',
  prival: 30,
  facilityval: 3,
  levelval: 6,
  facility: 'daemon',
  level: 'info',
  type: 'BSD',
  ts: 2018-08-04T14:53:13.000Z,
  host: 'pinger[1334]',
  message: '64 bytes from 87.250.250.242: seq=69 ttl=37 time=33.918 ms\n',
  chain: [],
  fields: [],
  header: '<30>Aug  4 16:53:13 pinger[1334]: ' }
```

**Comment by iatsiuk** [comment]
Many thanks!
